packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -149,6 +153,12 @@ const NewExperimentForm: FC<NewExperimentFormProps> = ({
   const settings = useOrgSettings();
   const { refreshWatching } = useWatching();
 
+  const { data: sdkConnectionsData } = useSDKConnections();
+  const hasSDKWithNoBucketingV2 = !allConnectionsSupportBucketingV2(
+    sdkConnectionsData?.connections,
+    project
+  );
```

Comment:
In the onboarding flow for experiments, we don't prompt them to create their first SDK connection until they are ready to start their experiment.  So `sdkConnectionsData?.connections` will be an empty array, the check below will fail, and it will set hashing to v1, which doesn't seem right.